lib/DBDefs/Default.pm

```diff
@@ -136,6 +136,9 @@ sub CANONICAL_SERVER          { "https://musicbrainz.org" }
 # The server used to link to CritiqueBrainz users and reviews.
 sub CRITIQUEBRAINZ_SERVER     { "https://critiquebrainz.org" }
 
+# The URL where static resources are located, excluding the trailing slash.
+sub STATIC_RESOURCES_LOCATION { 'http://localhost:5000/static' }
```

Comment:
Since this caused a problem already (IRC), perhaps it should be changed to use the `WEB_SERVER` definition? E.g.

    sub STATIC_RESOURCE_LOCATION { 'http://' . shift->WEB_SERVER . '/static' }

Perhaps even use `'//'` instead og `'http://'` so that it works for someone who has set up SSL, too?